O/oneTBB/build_tarballs.jl

```diff
@@ -1,20 +1,21 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
-using BinaryBuilder, Pkg
+using BinaryBuilder
 
 name = "oneTBB"
-version = v"2021.8.0"
+version = v"2021.9.0"
 
-# Collection of sources required to complete build
+# Collection of sources required to build hwloc
 sources = [
-    GitSource("https://github.com/oneapi-src/oneTBB.git",
-    "c9497714821c3d443ee44c732609eb6850195ffb"),
+    GitSource("https://github.com/oneapi-src/oneTBB", "a00cc3b8b5fb4d8115e9de56bf713157073ed68c"),
     DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/oneTBB*
+cd ${WORKSPACE}/srcdir/oneTBB
+
+rm -f /usr/share/cmake/Modules/Compiler/._*.cmake
```

Comment:
What exactly is the need to delete files included with the system CMake install?